app/components/LoadingIndicator/Circle.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled, { keyframes } from 'styled-components';
 
+/* stylelint-disable comment-empty-line-before */
```

Comment:
Is there a stylelint config? (sorry if I haven't checked the dev branch and there is)

Reply:
Yup it's in dev. You merged it in PR #2066 ;)

Reply:
Thank you 😅